config/crd/bases/mellanox.com_macvlannetworks.yaml

```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.10.0
```

Comment:
we really need to have some makefile targets to get a pre-defined version of controller-gen/operator-sdk so all generate with same version

Reply:
Here is a bug filed: https://github.com/Mellanox/network-operator/issues/444

Reply:
The version is defined here: https://github.com/Mellanox/network-operator/blob/master/Makefile#L248

The problem is that if you have an older version, it will not be updated unless you delete the older version from `./bin` directory.

Reply:
ack so it just making sure CRD are updated using up2date version. in that case i dont think we need the bug.